<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN166548" failures="1" tests="6" name="Test" time="61.317" errors="0" timestamp="2024-02-13T16:27:14 IST">
  <testcase classname="test.main" name="check" time="0.876"/>
  <testcase classname="test.main" name="count_news" time="0.05"/>
  <testcase classname="test.main" name="check_tt" time="0.309"/>
  <testcase classname="test.main" name="i_o_header" time="23.138">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [CEO blog: Elevating Cognizant’s stature on the world stage] but found [Our growth mindset]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [CEO blog: Elevating Cognizant’s stature on the world stage] but found [Our growth mindset]
at pages.home.i_o_headers(home.java:117)
at test.main.i_o_header(main.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- i_o_header -->
  <testcase classname="test.main" name="news_Execs" time="16.503"/>
  <testcase classname="test.main" name="apps_tools" time="2.389"/>
</testsuite> <!-- Test -->
